browser/net/brave_common_static_redirect_network_delegate_helper.cc

```diff
@@ -30,20 +26,21 @@ bool IsUpdaterURL(const GURL& gurl) {
                   std::string(component_updater::kUpdaterJSONDefaultUrl) + "*"),
        URLPattern(
            URLPattern::SCHEME_HTTP,
-           std::string(component_updater::kUpdaterJSONFallbackUrl) + "*"),
-       URLPattern(
+           std::string(component_updater::kUpdaterJSONFallbackUrl) + "*")
```

Comment:
it's actually fine (and clang format standard) to have a trailing comma here instead of putting it below